index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+
+</head>
+<body>
+    <header>
+        <h1>Hello Microverse!</h1>
+    </header>
```

Comment:
- [ ] As stated the project requirement Kindly make sure that A .css file that includes `one class` with styles for your header (make it in your favorite color).
- ![Capture](https://user-images.githubusercontent.com/93598189/211501647-d22fcab1-eabd-46f6-84a6-f12672abe6f9.PNG)